apps/explorer/src/ui/Button.tsx

```diff
@@ -13,13 +13,18 @@ const buttonStyles = cva(['inline-flex items-center justify-center relative'], {
         variant: {
             primary: 'bg-sui-dark text-sui-light hover:text-white border-none',
             secondary: 'bg-gray-90 text-gray-50 hover:text-white border-none',
+            tertiary:
+                'bg-white text-sui-dark hover:text-sui-darker border-none',
             outline:
                 'bg-white border border-steel text-steel-dark hover:text-steel-darker hover:border-steel-dark active:text-steel active:border-steel disabled:border-gray-45 disabled:text-steel-dark',
         },
         size: {
             md: 'px-3 py-2 rounded-md text-bodySmall font-semibold',
             lg: 'px-4 py-3 rounded-lg text-body font-semibold',
         },
+        padding: {
+            none: '!p-0',
```

Comment:
This is scary, not convinced that this is a valid use-case, why do you think you need this

Reply:
removed